setup.py

```diff
@@ -24,5 +34,9 @@
         "pillow",      # or PIL
         "PyWavelets",  # for whash
     ],
+    test_suite='imagehash.tests'
 )
 
+
+if __name__ == '__main__':
```

Comment:
Basically, previously `setuptools.setup` was being called at the module level. Which is fine. However, the disadvantage of this is that it **always** executes, even on `import` which you may not want to happen, particularly if a developer is changing `setup.py` and wants to see changes to the contents of the dictionary passed into `setuptools.setup`. Which is normal practice in several projects that I've seen.

I'm going to remove it, as it's clear to me that you want minimal changes. You've now seen this pattern and if you want it, you can easily re-implement it.